### PR TITLE
Laser pointer re-balance

### PR DESCRIPTION
**Pull request by ZombiesVsPlants**
## About The Pull Request

Reduces the paralyse time on silicons significantly and slightly increases the amount of charges and chance to hit/effect. 

## Why It's Good For The Game

A flash gives 7 seconds of paralyse.
The laser pointer was giving 10-20 seconds.

A robotics flash has 4 charges.
The laser pointer had 5.

A flash forces you to be adjacent to to your target.
The laser pointer can do it at range.

I feel like the laser pointer was quite overpowered and it has been reduced to something that can give you an edge instead of being the key in a silicon fight.

## Changelog
:cl:
balance: Changed Laser pointer to paralyse for a much shorter time than before.
balance: Increased the amount of charges a Laser pointer had by 1
balance: Increased the effectchance variable from 33 to 35
/:cl:

### CODE REVIEW

**Comment by MCterra10** [comment]
i ded

**Comment by ZombiesVsPlants** [comment]
First time so not sure what would be best, if you have any ideas for the values I would love to know

File: code/game/objects/items/devices/laserpointer.dm [hunk 1]
```diff
@@ -11,9 +11,9 @@
 	materials = list(/datum/material/iron=500, /datum/material/glass=500)
 	w_class = WEIGHT_CLASS_SMALL
 	var/turf/pointer_loc
-	var/energy = 5
-	var/max_energy = 5
-	var/effectchance = 33
+	var/energy = 6	\\Austation changed from 5 to 6
+	var/max_energy = 6 \\Austation changed from 5 to 6
+	var/effectchance = 35 \\Austation changed from 33 to 35
```
Comment (MCterra10) [inline code/game/objects/items/devices/laserpointer.dm:16]: bruh comments start with // not \\

File: code/game/objects/items/devices/laserpointer.dm [hunk 2]
```diff
@@ -121,7 +121,8 @@
 		//chance to actually hit the eyes depends on internal component
 		if(prob(effectchance * diode.rating))
 			S.flash_act(affect_silicon = 1)
-			S.Paralyze(rand(100,200))
+			\\different values on austation original values are rand(100,200)
```
Comment (MCterra10) [inline code/game/objects/items/devices/laserpointer.dm:124]: same here

Comment (ZombiesVsPlants) [reply code/game/objects/items/devices/laserpointer.dm]: done

File: code/game/objects/items/devices/laserpointer.dm (hunk 1 above)
Comment (ZombiesVsPlants) [reply code/game/objects/items/devices/laserpointer.dm]: done

**Comment by ZombiesVsPlants** [comment]
God Travis please

**Comment by Mick1299** [comment]
>S.Paralyze(rand(1 * diode.rating, 2 * diode.rating))

Rating returns either 1, 2, 3, or 4 depending on tier. You want to change a 100-200 (10-20 second) paralyse into a 4-8 (0.4-0.8 second) paralyse with the highest tier of microlaser. That isn't a nerf, that guts the laser pointers only use

**Comment by MCterra10** [comment]
laser pointers are a roleplay item change my mind

**Comment by Mick1299** [comment]
We're NRP, roleplay is banned here

**Comment by TheFakeElon** [comment]
an upgraded laser is one of the best ways to counter a good secborg. This basically only leaves us with EMPs. While flashes are good on most borgs they're not very effective against secborgs due to their ranged options. Silicons seem to get buffed with every PR. I'm personally against this until sekboogs get a nerf to their disabler/overall capabilities.

If you want to ignore that comment at least take into consideration that a maxed laser pointer should at the **very** least provide a comfortable amount of time for the user to pull out a flash and use it on the stunned borg.

salt pr btw

**Comment by Mick1299** [comment]
I'm glad someone else sees it. 

make it something around S.Paralyze(rand(3 * diode.rating, 6 * diode.rating))

That would be a 1-2.5 second stun with t4 parts

**Comment by TheFakeElon** [comment]
2.5 seconds sounds _relatively_ fair I guess. Although I still firmly believe that this change isn't needed at all (not that I have a say in it lmao)

**Comment by ZombiesVsPlants** [comment]
WAIT FUCK I FUCKED IT DOn'T MERGE

**Comment by ZombiesVsPlants** [comment]
Ok fixed it now it can merge

**Comment by MCterra10** [comment]
I'm happy with that. Also bee just merged the fix to using them through walls, so this looks good.

**Comment by ZombiesVsPlants** [comment]
H E L L   Y E A H

**Pull request merged**